### ISSUE TITLE
update tables in option page to have link to exact line of table

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#9070]
https://checkstyle.org/property_types.html#LeftCurlyOption

it has bunch of options, sharing a link to table still not precise location target user should read.
I want to have a link to `nlow`. 
We already practice exact links to table content at https://checkstyle.org/google_style.html
it would be awesome to have the same links in page property_types.html for all tables.

NOTE: if we like this .... we might update all tables in Check documentation https://checkstyle.org/config_blocks.html#LeftCurly_Properties to make link to exact property.

**Comment by romani** [comment]
@rnveach , @pbludov , @strkkk , @nmancus1 , @esilkensen 

please share your thought on this issue and vote on it to approve or not.

**Comment by pbludov** [comment]
I like the idea. Also, the "since" column has plain text now. There can be a link to the release.

**Comment by esilkensen** [comment]
Agreed, I like it

**Comment by nmancus1** [comment]
This is a good idea, I was thinking of something like this the other day when reading the documentation for LeftCurlyCheck, but take it a bit further.  I think it may be good to list the options with a link to each option.  It wouldn't take up additional space in the table, and once you are familiar with the abbreviations, you would only have to look at the table in LeftCurlyCheck to be reminded of the options, and never even have to visit the LeftCurlyOptions table link.

I think @pbludov 's idea of linking to the release notes is good, too.  Here is an image of what I had in mind:
![image](https://user-images.githubusercontent.com/31252532/100758216-41c1fd80-33bd-11eb-9de5-89933bc74b34.png)

**Comment by romani** [comment]
> Also, the "since" column has plain text now. There can be a link to the release.

lets keep it as separate update.
There is a problem with releasenotes page .... it is become to big already, and I need to move old releases to https://checkstyle.org/releasenotes_old.html but this will damage all links. I still have no idea how to resolve this problem.

**Comment by romani** [comment]
>  I think it may be good to list the options with a link to each option.

good idea, It will require some coding to teach our code to enforce it by unit test. 
Lets keep it as separate idea/issue.

**Comment by strkkk** [comment]
I am ok with link

**Comment by rnveach** [comment]
> There is a problem with releasenotes page .... it is become to big already, and I need to move old releases

Some other websites make a separate page for each version. With that method, all version numbers would have their own link and would never go bad.

Release page could just list short summary of all releases and allow users to click for more details.

If 1 page is too much for a single release, group them by 2 or 3.

**Comment by Vyom-Yadav** [comment]
@rnveach so if I am getting it right, you want links for property name in the table just like there is a link for various tokens like `LITERAL_IF`, etc.

**Comment by Vyom-Yadav** [comment]
@strkkk ping
